error_handling_on_chain/README.md

```diff
@@ -0,0 +1,127 @@
+# Error Handling On Chain
+
+This example shows you how to handle errors in Blocky AS function calls on chain.
+It builds on the [Hello World On Chain](../hello_world_on_chain) example,
```

Comment:
We are also building on the other error handling example.

While we are on the topic, could you rename `error_handling` to `error_handling_attest_fn_call` to keep the naming parallelism?

Reply:
Updated `error_handling` to `error_handling_attest_fn_call` and all references to it